src/ex12_html_css_js/task_01.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Task_01</title>
+    <link type="text/css" rel="stylesheet" href="task_01.css">
+</head>
+<body>
+
+<div class="wrapper">
```

Comment:
make more obvious namings for the root block container

Reply:
Like what? The wrapper block is a wrapper for the entire site and not for a specific block.

Reply:
ok then, might be wrapper is ok, but you could also use HTML5 tags like main (instead of wrapper), article (instead of container), for mentioned blocks

Reply:
not argue. replace the wrapper class with the corresponding tag

Reply:
haven't you pushed changes, have you?